packages/semantic-ui/src/components/List.js

```diff
@@ -331,6 +351,36 @@ const useList = (WrappedComponent: ComponentType<any>) => (
       this.setState({ selectedItem: copy, modalEdit: true });
     }
 
+    /**
+     * Generates and downloads a CSV file containing all
+     * the data in the table.
+     *
+     * @param items
+     */
+    onCsvExportButton() {
+      let csv = `${this.props.columns.map((col) => `"${col.label}"`).join(',')}\n`;
+
+      this.props.items.forEach((item) => {
```

Comment:
One of the benefits to the `underscore` library functions is that they are null safe. I would recommend either using the `underscore` functions (which is consistent with the rest of the component) or adding explicit `null`/`undefined` checks to avoid errors.